modules/dateUtils.js

```diff
@@ -0,0 +1,10 @@
+export default function getCurrentDate() {
+  const options = {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
```

Comment:
Nice work so far. However, as pointed out by the previous reviewer, kindly consider using the arrow function for this one too since it is a project requirement. Kindly consider converting this and all other functions to the modern ES6 arrow function. :wink: